modules/tinymce/src/themes/silver/main/ts/ui/dialog/WindowManager.ts

```diff
@@ -122,6 +122,7 @@ const setup = (extras: WindowManagerSetup): WindowManagerImpl => {
         backstage
       );
 
+      editor.ui.show();
```

Comment:
This isn't the right place for this, if you read the Jira comments I had mentioned it should go in the **core** WindowManager, this is in the theme and only solve the issue for modal dialogs not inline dialogs, which is why it should be in core.

Reply:
oh I missed that it was in the wrong place. good pickup.

Reply:
Looking, I think somewhere in the `storeSelectionAndOpenDialog` function is probably going to be best.